examples/jsm/loaders/FBXLoader.js

```diff
@@ -858,8 +858,6 @@ class FBXTreeParser {
 
 		this.createAmbientLight();
 
-		this.setupMorphMaterials();
```

Comment:
@looeee It seems this method is not necessary anymore since materials can now be shared across normal and morphed 3D objects.

Reply:
Yup, agreed.